Project-Euler/Project-Euler/Program.cs

```diff
@@ -20,19 +37,61 @@ public static void PrintIntro()
             //Add a couple more lines, maybe?
         }
 
-        public static void PrintMenuOptions() //This method will be private once I write the method that will call it. blah blah
+        public static int GetMenuSelection()
+        {
+            PrintMenuOptions();
+
+            while (true)
+            {
+                string userSelection = Console.ReadLine().ToUpper();
```

Comment:
If the user ever just hits <Enter> without typing anything, I think _Console.ReadLine()_ will return null. Which will blow up on _.ToUpper()_. To fix this add a question mark before _.ToUpper();_ (Google what a question mark before a method call will do)